### PR TITLE
Entity vitals

### PR DESCRIPTION
**Pull request by EMcQ-BES**
### Changes:

- Add reports for lesmis

### CODE REVIEW

File: packages/database/src/migrations/20210412210749-AddLesmisEntityVitalsReports-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+      },
+      transform: [
+        'keyValueByDataElementName',
+        'mostRecentValuePerOrgUnit',
+        {
+          transform: 'select',
+          '...': ['organisationUnit'],
+          "'Photo'": '$row.SchFDpho',
+          "'ContactNumber'": '$row.SchCVD026b',
+          "'NumberOfStudents'":
```
Comment (edmofro) [inline packages/database/src/migrations/20210412210749-AddLesmisEntityVitalsReports-modifies-data.js:74]: I reckon this should be an indicator

File: packages/database/src/migrations/20210412210749-AddLesmisEntityVitalsReports-modifies-data.js [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+          'SchCVD026b',
+          'SchFDpho',
+        ],
+      },
+      transform: [
+        'keyValueByDataElementName',
+        'mostRecentValuePerOrgUnit',
+        {
+          transform: 'select',
+          '...': ['organisationUnit'],
```
Comment (edmofro) [inline packages/database/src/migrations/20210412210749-AddLesmisEntityVitalsReports-modifies-data.js:71]: Remind me what this does?

**Review by edmofro — CHANGES_REQUESTED** [review]
lgtm other than making the sum of students an indicator

File: packages/database/src/migrations/20210412210749-AddLesmisEntityVitalsReports-modifies-data.js (hunk 2 above)
Comment (EMcQ-BES) [reply packages/database/src/migrations/20210412210749-AddLesmisEntityVitalsReports-modifies-data.js:71]: This just includes the passed in org unit in the results of the report, probably not entirely necessary for our usage

Comment (edmofro) [reply packages/database/src/migrations/20210412210749-AddLesmisEntityVitalsReports-modifies-data.js:71]: Aha :+1:

**Review by edmofro — APPROVED** [review]
All looking good to me now! So long as you've tested those suggested changes I think this is good to go :-)

**Comment by EMcQ-BES** [comment]
Yep 👍  Ran a quick test in postman requesting one of the school vitals reports

image

**Pull request merged**